sdgx/data_models/relationship.py

```diff
@@ -16,8 +18,12 @@ class Relationship(BaseModel):
     parent_table: str
     child_table: str
 
-    # keys
-    child_table_foreign_key: str
+    # foreign keys
+    child_table_foreign_key: str = "foreign key undefined"
+
+    # for composite keys
+    composite_foreign_key: bool = False
+    child_table_composite_foreign_key: List[str] = []
```

Comment:
Keep it same as `Metadata.primary_keys`

```suggestion
    foreign_keys: List[str] = []
```